install-dev/upgrade/sql/1.7.0.0.sql

```diff
@@ -19,17 +25,11 @@ ALTER TABLE PREFIX_customer ADD `reset_password_validity` datetime DEFAULT NULL;
 ALTER TABLE PREFIX_employee ADD `reset_password_token` varchar(40) DEFAULT NULL;
 ALTER TABLE PREFIX_employee ADD `reset_password_validity` datetime DEFAULT NULL;
```

Comment:
Can you add before, because it make an error : 
ALTER TABLE `PREFIX_employee` CHANGE `last_connection_date` `last_connection_date` DATE NULL DEFAULT NULL;

![sql_2](https://cloud.githubusercontent.com/assets/9074610/19469541/0f363154-951c-11e6-9ee5-0229cff53aea.png)